internal/commands/util/pr.go

```diff
@@ -283,6 +328,108 @@ func runPRDecorationGitlab(prWrapper wrappers.PRWrapper, policyWrapper wrappers.
 	}
 }
 
+func runPRDecorationBitbucket(prWrapper wrappers.PRWrapper, policyWrapper wrappers.PolicyWrapper, scansWrapper wrappers.ScansWrapper) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		scanID, _ := cmd.Flags().GetString(params.ScanIDFlag)
+		scmTokenFlag, _ := cmd.Flags().GetString(params.SCMTokenFlag)
+		namespaceFlag, _ := cmd.Flags().GetString(params.NamespaceFlag)
+		repoNameFlag, _ := cmd.Flags().GetString(params.RepoNameFlag)
+		prIDFlag, _ := cmd.Flags().GetInt(params.PRBBIDFlag)
+		apiURL, _ := cmd.Flags().GetString(params.CodeRepositoryFlag)
+		projectKey, _ := cmd.Flags().GetString(params.ProjectKeyFlag)
+
+		isCloud, flagRequiredErr := checkIsCloudAndValidateFlag(apiURL, namespaceFlag, projectKey)
+		if flagRequiredErr != nil {
+			return flagRequiredErr
+		}
+
+		scanRunningOrQueued, err := IsScanRunningOrQueued(scansWrapper, scanID)
+
+		if err != nil {
+			return err
+		}
+
+		if scanRunningOrQueued {
+			log.Println(noPRDecorationCreated)
+			return nil
+		}
+
+		policies, policyError := getScanViolatedPolicies(scansWrapper, policyWrapper, scanID, cmd)
+		if policyError != nil {
+			return errors.Errorf(policyErrorFormat, failedCreatingBitbucketPrDecoration)
+		}
+
+		prModel := createPRModel(isCloud, scanID, scmTokenFlag, namespaceFlag, repoNameFlag, prIDFlag, apiURL, projectKey, policies)
+		prResponse, errorModel, err := prWrapper.PostPRDecoration(prModel)
+
+		if err != nil {
+			return err
+		}
+
+		if errorModel != nil {
+			return errors.Errorf(errorCodeFormat, failedCreatingBitbucketPrDecoration, errorModel.Code, errorModel.Message)
+		}
+
+		logger.Print(prResponse)
+		return nil
+	}
+}
+
+func repoSlugFormatBB(repoNameFlag string) string {
+	repoSlug := strings.Replace(repoNameFlag, " ", "-", -1)
+	return repoSlug
+}
+
+func checkIsCloudAndValidateFlag(apiURL, namespaceFlag, projectKey string) (bool, error) {
```

Comment:
Modified the repoSlugFormatBB function to
	"repoSlug := strings.Replace(strings.ToLower(repoNameFlag), " ", "-", -1)"
	because we need also to converts the entire string to lowercase for repoSlug
	e.x from ‘check Pr Decorations’  to  ‘check-pr-decorations’

Reply:
That's not necessary, I've checked it.